rdkit/Chem/UnitTestDescriptors.py

```diff
@@ -162,18 +162,19 @@ def testVectorDescriptors(self):
       f = getattr(Descriptors, n)
       self.assertEqual(results[i], f(m))
 
-    def testVectorDescriptorsInDescList(self):
-      # First try only bcuts should exist
-      descriptors = set([n for n,_ in Descriptors.descList])
-      names = set(["BCUT2D_%s"%str(i+1) for i in range(8)])
-      self.assertEqual(descriptors.intersection(names), names)
-      
-      Descriptors.setupAUTOCORRDescriptors()
-      descriptors2 = set([n for n,_ in Descriptors.descList])
-      self.assertEqual(descriptors2.intersection(descriptors), descriptors)
-
-      names = ["AUTOCORR2D_%s"%str(i+1) for i in range(192)]
-      self.assertEqual(descriptors.intersection(names), names)
+  def testVectorDescriptorsInDescList(self):
+    # First try only bcuts should exist
+    descriptors = set([n for n,_ in Descriptors.descList])
+    names = set(["BCUT2D_%s"%i for i in ('MWHI',"MWLOW","CHGHI","CHGLO",
```

Comment:
This is code breaking it looks like

Reply:
As it changes the function names in the descriptor list.

Reply:
Ok, reformatting it.

Reply:
Thanks, I probably shouldn't have copied the AUTOCORR format, but that's what I did :)

Reply:
@ricrogz - sorry I misread the change!  It was fine.

Reply:
No problem! I wasn't quite sure whether you complained about the formatting, which was ugly, or whether the names of the descriptors were wrong... which was part of the problem here.